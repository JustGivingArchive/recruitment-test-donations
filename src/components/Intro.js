import React from 'react';

const Intro = () => {
  return (
    <div className="main_container">
      <h2 className="subtitle">
        Please select a charity to see donations.
      </h2>
    </div>
  );
};

export default Intro;
