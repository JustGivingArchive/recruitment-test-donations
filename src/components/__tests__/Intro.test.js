import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../Intro.js';

describe('Test Intro component', () => {

  it('matches snapshot', () => {
    const tree = renderer.create(
      <Intro />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})
