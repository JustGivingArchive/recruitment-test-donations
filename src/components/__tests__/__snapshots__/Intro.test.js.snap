exports[`Test Intro component matches snapshot 1`] = `
<div
  className="main_container">
  <h2
    className="subtitle">
    Please select a charity to see donations.
  </h2>
</div>
`;
